docs/configure/overview.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Overview'
+---
+
+Storybook is configured via a folder, called `.storybook` which contains various configuration files. 
+
+> Note you can change the folder that Storybook uses by setting the `-c` flag to your `start-storybook` and `build-storybook` scripts.
+
+
+### Configure your Storybook project
+
+The main configuration file is `main.js`. This file controls the behaviour of the Storybook server, and so you must restart Storybook’s process when you change it. It contains the following:
+
+```js
+// .storybook/main.js
+
+module.exports = {
+  stories: ['../src/**/*.stories.(js|mdx)'],
+  addons: ['@storybook/addon-essentials']
+}
+```
+
+The `main.js` configuration file is a [preset](../presets/introduction) and as such has a powerful interface, but the key fields within it are:
```

Comment:
I'm not sure we are keeping this link.. perhaps it should link to the preset section of the api/addons section?

Reply:
let me know if the link works for you